src/components/common/Profile.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+  <div class="max-w-sm">
+    <div class="flex w-full p-2">
+      <div class="group relative">
+        <p
+          class="
+            w-24
+            text-center
+            opacity-0
+            absolute
+            top-1/2
+            left-1/2
+            transform
+            -translate-x-1/2 -translate-y-1/2
+            group-hover:opacity-100
+          "
+        >
+          Edit profile
+        </p>
+        <Avatar
+          @click="buttonFunction"
+          :firstName="name"
+          class="group-hover:opacity-50"
+        />
+      </div>
+      <div class="flex flex-col justify-center ml-3">
+        <p class="font-medium text-5xl break-all">
+          {{ name }}
+        </p>
+        <p class="text-2xl break-all">{{ surname }}</p>
+      </div>
+    </div>
+    <div class="flex flex-col p-2">
+      <p class="">Podstawowe informacje:</p>
+      <p>
+        Waga: <span class="font-semibold">{{ weight }} kg</span>
```

Comment:
z tego co widziałem designy to user jest w stanie zmienić sobie jednostkę profilu, więc to nie powinno być statyczne. zapisz najlepiej do stora jakich używa, może jakiś mały komponent co automatycznie konwertuje